src/Tribe/Repositories/Attendee/RSVP.php

```diff
@@ -14,6 +18,37 @@ class Tribe__Tickets__Repositories__Attendee__RSVP extends Tribe__Tickets__Atten
 	 */
 	protected $key_name = 'rsvp';
 
+	/**
+	 * {@inheritdoc}
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->attendee_provider = tribe( 'tickets.rsvp' );
+
+		$this->create_args['post_type'] = $this->attendee_provider->attendee_object;
+
+		// Use a regular variable so we can get constants from it PHP <7.0
+		$attendee_provider = $this->attendee_provider;
+
+		// Add object specific aliases.
+		$this->update_fields_aliases = array_merge( $this->update_fields_aliases, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 2f313659984ea909242e83fe02b2b595</sub>